public/usage-examples/graphics/fill_ellipse/fill_ellipse-2-extended-oop.cs

```diff
@@ -0,0 +1,23 @@
+using SplashKitSDK;
+
+namespace Program
+{
+    public class Program
+    {
+        public static void Main()
+        {
+            SplashKit.OpenWindow("Clover Drawing with Fill Ellipse", 1000, 600);
+
+            SplashKit.ClearScreen();
+            SplashKit.FillEllipse(SplashKit.ColorGreen(), 150, 225, 400, 200);
+            SplashKit.FillEllipse(SplashKit.ColorGreen(), 375, 0, 200, 400);
+            SplashKit.FillEllipse(SplashKit.ColorGreen(), 400, 225, 400, 200);
+            SplashKit.FillRectangle(SplashKit.ColorBrown(), 470, 400, 10, 150);
```

Comment:
Just one other thing, here you should be using the OOP version for colours.
So for example it would be `SplashKit.FillEllipse(Color.Green, 150, 225, 400, 200);`